.github/workflows/deploy-infra.yaml

```diff
@@ -121,6 +135,7 @@ jobs:
         run: |-
           yq w infra/cluster.yaml metadata.name "biomage-$CLUSTER_ENV" | yq w - metadata.region ${{ env.REGION }} > /tmp/cluster-$CLUSTER_ENV.yaml
           cat /tmp/cluster-$CLUSTER_ENV.yaml
+          echo /tmp/cluster-$CLUSTER_ENV.yaml
```

Comment:
what's the difference between this `echo` and the `cat` above? Arent' they showing the same thing?

Reply:
echo prints the name, cat prints the contents. I will delete this line, it was only for debugging purposes.